wporg-two-factor.php

```diff
@@ -226,6 +236,22 @@ function get_enable_2fa_notice( string $existing_notices = '' ) : string {
 	return $two_factor_notice . $existing_notices;
 }
 
+/*
+ * Remove the 2FA settings page from the admin menu.
+ *
+ * We don't want site admins making changes, etc.
+ */
+function block_webauthn_settings_page( ) {
```

Comment:
Looks like an unintentional space between the parenthesis?

Reply:
fixed 👍🏻